defaults/k8s_eks_config.yaml

```diff
@@ -35,7 +35,7 @@ k8s_scylla_disk_class: 'local-raid-disks'
 # To test nightly builds define it like this: 'scylladb/scylla-operator:nightly'
 k8s_scylla_operator_docker_image: ''
 k8s_scylla_operator_helm_repo: 'https://storage.googleapis.com/scylla-operator-charts/latest'
```

Comment:
the repo should be stable
https://storage.googleapis.com/scylla-operator-charts/stable/

Reply:
Done